Sources/TuistSupport/SwiftPackageManager/PackageInfo.swift

```diff
@@ -497,7 +497,7 @@ extension PackageInfo.Target.TargetType {
     /// Defines if the target would be processed when processing the package
     public var isSupported: Bool {
         switch self {
-        case .regular, .system:
+            case .regular, .system, .test:
```

Comment:
This should return true only if configured to do so, to avoid processing targets if not needed